lib/djvViewApp/PlaybackSpeedWidget.cpp

```diff
@@ -148,6 +166,46 @@ namespace djv
                         }
                     }
                 });
+
+            p.useCustomSpeedButton->setClickedCallback(
+                [weak]
+                {
+                    if (auto widget = weak.lock())
+                    {
+                        widget->_doSpeedCallback(widget->_p->customSpeed);
+                    }
+                });
+
+            p.customSpeedLineEdit->setTextEditCallback(
+                [weak](const std::string& value, UI::TextEditReason)
+                {
+                    if (auto widget = weak.lock())
+                    {
+                        {
+                            try
+                            {
+                                if (std::stof(value) >= 1) 
+                                {
+                                    //widget->_doSpeedCallback(Time::Speed(std::stof(value)));
+                                    widget->_p->customSpeed = Time::Speed(std::stof(value));
+                                }
+                                else if (std::stof(value) > 0.F && std::stof(value) < 1.F)
+                                {
+                                    //widget->_doSpeedCallback(Time::Speed(std::floor(std::stof(value) * 1000), 1000));
+                                    widget->_p->customSpeed = Time::Speed(std::floor(std::stof(value) * 1000), 1000);
+                                }
+                            }
+                            catch (const std::exception&)
+                            {
+                                std::stringstream ss;
+                                ss << "Cannot parse the value.";
+                                widget->_log(ss.str(), LogLevel::Error);
+                            }
+                        }
```

Comment:
Not a big deal, but it looks like this extra "{}" block around the try/catch isn't necessary.